### DISCUSSION TITLE
MONAI

### DISCUSSION THREAD

**Discussion by akraskovSMDB** [neuropoly/axondeepseg#753] [Help needed]
Are there any timelines for monai instead of ivadomed as an alternative for the backbone of ADS?

**Comment by hermancollin** [comment]
@akraskovSMDB unfortunately there is no timeline. Moreover it's unlikely we will be switching to a monai backend.

What do you think of a zoom call to discuss your specific needs? I might have suggestions for your project. I remember you mentioned training your own model earlier this year.

**Reply by hermancollin** [reply]
Well actually we are considering MONAI as an option, maybe my previous statement was a bit strong. We have other prospects for now but MONAI has some advantages.

Why were you asking this question? Were you considering trying it yourself?

**Reply by akraskovSMDB** [reply]
I thought that the reason to switch to monai was because ivadomed is not being developed, I guess you are considering other alternatives.

Zoom would be great. Could you please let me know your availability, thank you

**Reply by hermancollin** [reply]
I am available for a call anytime Thursday, Friday or next week (timezone UTC-4). You can answer your availability by sending an email to axondeepseg@googlegroups.com and I will send you a zoom link.
